docs/reference/plugins/python.rst

```diff
@@ -0,0 +1,45 @@
+Python plugin
+-------------
+
+The Python plugin can be used for Python projects where you would want to do
+any of the following things:
+
+- Import Python modules with a :file:`requirements.txt` file.
+- Build a Python project that has a :file:`setup.py` or
+  :file:`pyproject.toml` file.
+- Install packages using :command:`pip`.
+
+This plugin uses the common plugin keywords as well as those for "sources".
+For more information check the 'plugins' topic for the former and the
+'sources' topic for the latter.
+
+Additionally, this plugin uses the following plugin-specific keywords:
```

Comment:
IDK whether this is the right time to do this, but this document will probably be a good spot to include a sample YAML segment